play-json/src/main/scala/com/lightbend/lagom/scaladsl/playjson/JsonMigration.scala

```diff
@@ -74,7 +86,13 @@ object JsonMigrations {
  * class name you should override [[transformClassName]] and return
  * current class name.
  */
-abstract class JsonMigration(val currentVersion: Int) {
+abstract class JsonMigration(val currentVersion: Int, val supportedForwardVersion: Int) {
+
+  def this(currentVersion: Int) = this(currentVersion, currentVersion)
+
+  require(
+    currentVersion <= supportedForwardVersion,
+    """The "currentVersion" of a JacksonMigration must be less or equal to the "supportedForwardVersion".""")
```

Comment:
```suggestion
    s"""The "currentVersion"=$currentVersion of a JacksonMigration must be less or equal to the "supportedForwardVersion"=$supportedForwardVersion.""")
```

Reply:
I copy/pasted that and it says `JacksonMigration` aswell. 🤦🏼‍♂️